apps/ledger-live-desktop/package.json

```diff
@@ -48,8 +48,8 @@
     "test:playwright:clean": "git clean -fdX tests/artifacts",
     "typecheck": "node scripts/typecheck.js",
     "check": "pnpm lint; pnpm typecheck",
-    "unimported": "unimported",
-    "assets:replace": "zx ./scripts/replace-assets.mjs"
+    "assets:replace": "zx ./scripts/replace-assets.mjs",
+    "knip-check": "pnpm knip --directory ../.. --workspace apps/ledger-live-desktop"
```

Comment:
[ASK] is it necessary to use workspace tracking here since you are already in the correct workspace ?

Reply:
Yep need that otherwise knip doesn't understand in which workspace we are and will trigger the test for every lib. In the docs they mentioned that the wanted workspace must be in the cli argument to trigger it correctly

Reply:
The --directory allows us to use the common file directory by running the command from this directory. For example, if we want to keep a customized knip configuration in a specific library, we won't need this option. And --workspace specifies which entry the test should take in the configuration

Reply:
Gotchu 👍

Reply:
Hmm not sure about that because it's using pnpm-workspace.yaml (I've tried it doesn't work).IMO the only way would be to have a config for every lib that can extend from a baseConfig (so we don't centralise everything at root using knip workspace as it's done here). But if we want to keep the workspace part we need to use these arguments. I can at least reduce them with -W and -D.
It only depends on how we want to implement it actually we can keep the logic to have a config file for each lib that extends a knip.config.js with only the rules but no entry point etc or we keep the workspace part. A cool thing is that we can mix both

Reply:
Alright, makes sense. 
As a note, all the other tools in the monorepo work the way I described (a based config at root + extend in each project).
This is non blocking however and good to go this way, I just wanted to understand the options we had.

Thanks for the informations 🙏